inst/tests/nafill.Rraw

```diff
@@ -149,8 +149,15 @@ test(4.20, colnamesInt(dt, integer()), integer())
 test(4.21, colnamesInt(dt, NULL), seq_along(dt))
 test(4.22, colnamesInt("asd", 1), error="must be data.table compatible")
 test(4.23, colnamesInt(dt, 1, check_dups="a"), error="check_dups")
+test(4.24, colnamesInt(dt, c("a", "e"), skip_absent = TRUE),as.integer(c(1,0)))
```

Comment:
You can use `INT(1, 0)`, though I personally prefer `c(1L, 0L)`.

Also, for data.table style, use `skip_absent=TRUE`.